### PR TITLE
KHADRI-190924-1035

### PR DESCRIPTION
**Pull request by thasmiyas39**

### CODE REVIEW

**Comment by thasmiyas39** [comment]
conflicts are resolved

**Review by roohishaik128 — APPROVED** [review]

**Comment by thasmiyas39** [comment]
Resolved

File: src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java [hunk 1]
```diff
@@ -33,8 +37,30 @@ public void insertData(Mobile mobile) {
 			System.out.println("MobileInsertDao Exception occours:" + e);
 		} finally {
 			JdbcConnectionUtil.closeResources();
+
+			System.out.println("MobileInsertDao : MobileInsertDao(-) ends");
+
+		}
+	}
+
+	public void mobileSelectData() {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java]: change from mobileSelectData to selectData.

File: src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java:1]: Change the name from MobileInsertDao to MobileDao

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeSelectDao.java:1]: dont add this file.

File: src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java (hunk 1 above)
Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java]: changed to selectData

File: src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java
Comment (thasmiyas39) [reply src/main/java/com/khadri/jdbc/prepare/data/mobile/dao/MobileInsertDao.java:1]: changed to MobileDao

**Comment by thasmiyas39** [comment]
Processed the files

**Comment by thasmiyas39** [comment]
Now there are no conflicts

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**